wasm.rkt

```diff
@@ -1281,3 +1290,23 @@
                   (call my_func2)))
            (term (const 7918331)) #:trace #f)
 
+;; If 0
+(test-wasm (term ((module
+                      (func my_func
+                            (const 0)
+                         (if (const 10) else (const 20))
```

Comment:
nit: indentation? Is this how dr racket would indent it? seems like the `(if` should be aligned with the `(const 0)` above (or the const should be moved to the left)